article-image_segmentation_1/article.md

```diff
@@ -0,0 +1,573 @@
+
+Image segmentation tasks can be classified based on the amount and type of information they convey. Below is an overview of these approaches, followed by visual outputs for each segmentation type.
+
+| Type                  | Goal                                         | Input                                                  | Output                                                                                                                           |
+| --------------------- | -------------------------------------------- | ------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
+| Instance Segmentation | Find every distinct object of target classes | An image and some instance target classes              | A bounding box and a segmentation mask for each instance of target classes                                                       |
+| Semantic Segmentation | Classify every pixel on an image             | An image and some semantic target classes              | Pixel-perfect segmentation map of the whole image                                                                                |
+| Panoptic Segmentation | Combine Instance and Semantic Segmentation   | An image and some instance and semantic target classes | Pixel-perfect segmentation map of the whole image AND a bounding box and a segmentation mask for each instance of target classes |
+
+|       ![Original Image](images/street_og.png)        | ![Semantic Segmentation](images/street_semantic.png) |
+| :-------------------------------------------------:  | :--------------------------------------------------: |
+|                  *Original Image*                    |              *Semantic Segmentation*                 |
+| ![Instance Segmentation](images/street_instance.png) | ![Panoptic Segmentation](images/street_panoptic.png) |
+|              *Instance Segmentation*                 |              *Panoptic Segmentation*                 |
+
+Each segmentation type possesses distinct traits and finds relevance in diverse applications. In the subsequent section, we will explore some common image segmentation techniques with examples, showcasing their respective strengths and weaknesses.
+
+## Image Segmentation Techniques <a id="techniques"></a>
+
+Over the years, the field of image segmentation has witnessed the development of various techniques, broadly classified into two main categories: traditional methods and deep learning approaches, as depicted in Figure 3. While both categories offer valuable insights, this article will focus exclusively on traditional techniques. The discussion of deep learning methods will follow in Part 2.
+
+![Image Segmentation Techniques](images/techniques.png)
+
+*Figure 3. Image Segmentation Techniques. (I coded this diagram using Mermaid; could it be designed by someone in the design team?)*
```

Comment:
Could anyone from the design team help to create such a diagram or shall I do it myself? The current one doesn't look good.

Reply:
Yes, that could be done with the help of someone from the design team, I will reach out to them and let them know about it, thx for mentioning it!